deploy/charts/monitoring/templates/networkpolicy.yaml

```diff
@@ -12,6 +12,8 @@ spec:
       protocol: TCP
     - port: 8443
       protocol: TCP
+    - port: 9090
+      protocol: TCP
```

Comment:
I think, we also use 9091 and even 9092 in some cases. Those are mainly for Collector that may have 3 pods and for the case where we consume metrics over https (must be enabled). 
It is not a blocker for merging this, but I thought that we may include it here for the sake of completeness.

Reply:
Ack, this is not the right network policy. Sorry for the confusion.